### ISSUE TITLE
Dropdown menus are broken on the VideoAG website

### ISSUE THREAD

**Issue by UnlimitedCookies** [Rc-Cookie/quality-of-rwth#8]
When clicking on a dropdown menu an error shows in the browser console. The error does not appear, when the extension is disabled.

One weird thing I noticed: When you delete all cookies and then reload the site, you are able to open the dropdown menu once, before you get the same error message all over again.

Here is the backtrace:
```
registerActiveTab rwthTools.js:147:17
videoAGAutoLoginForward rwthTools.js:147:17
Some cookies are misusing the recommended “SameSite“ attribute 2
Uncaught Error: Permission denied to access property "apply"
    jQuery 5
    toggle https://video.fsmpi.rwth-aachen.de/static/bootstrap/bootstrap.js?v=a42d9025797a0a169255ecc1ca344723f82dbd49:833
    jQuery 8
    <anonymous> https://video.fsmpi.rwth-aachen.de/static/bootstrap/bootstrap.js?v=a42d9025797a0a169255ecc1ca344723f82dbd49:170
    <anonymous> https://video.fsmpi.rwth-aachen.de/static/bootstrap/bootstrap.js?v=a42d9025797a0a169255ecc1ca344723f82dbd49:172
jquery.js:8036:9
```

**Comment by Rc-Cookie** [comment]
Does the dropdown open despite the error / is there anything actually not working, ignoring the error message itself?

On which exact video AG page are you experiencing the issue? I am unable to replicate the error and have tried video.fsmpi.rwth-aachen.de and rwth.video, and for each both the main page and a video playback.

This error does sound kind of weird to me though anyways, because I don't use jQuery (and also cannot access the instance loaded by the main page).

**Comment by UnlimitedCookies** [comment]
Yes, the menu cannot be opened (that's the bug). It does appear for example on https://video.fsmpi.rwth-aachen.de/22ws-buk/16858.
This might be a firefox related issue. And jquery is used by the VideoAG, not your extension :D.

Same bug on rwth.video.

**Comment by Rc-Cookie** [comment]
I am using Firefox, too, but I can't reproduce error. I can open the dropdown just fine both with and without the video playing, and there aren't any such errors either.

If the dropdown doesn't open, there is probably an error message for it, too. You can display the console of the popup window using `Extensions > Manage Extensions > Settings Icon > Debug add-ons > Quality of RWTH > Inspect > Console`. Maybe you can test and see whether there are any errors logged when you try to open the dropdown.

Other than that I can only suggest you check whether this might be a conflict with another extension you have installed.

**Comment by UnlimitedCookies** [comment]
There are no errors logged unfortunately :/
